db/V4.8_CREATE_PROCEDURE_FILTERING_USERS.sql

```diff
@@ -0,0 +1,23 @@
+IF EXISTS (SELECT * FROM sys.objects WHERE type = 'P' AND name ='FILTERING_USERS')
```

Comment:
@usyktt I'm not seeing definition of this procedure on changelogfile

Reply:
@olegRakus okay, it should be in this way
at this moment, we're not using this procedure
firstly, I'll write front-end part and then will say to Tanya if procedure and controller needs to be changed